pyproject.toml

```diff
@@ -55,7 +55,13 @@ dependencies = [
 
 [project.optional-dependencies]
 dev = [
+    "black",
+    "flake8",
+    "freezegun",
+    "isort",
     "pre-commit",
+    "pylint",
+    "pylint-django",
```

Comment:
same comment as above.

Reply:
Probably the same issue as above. What should it be set to?

Reply:
I'll try copying down the pyproject.toml from main, add my new dev dependency, and re-gen the lock files. 

I'm really confused though - I just pulled from main, so I don't know why I got all of those merge conflicts.

Reply:
https://github.com/chicagopython/chipy.org/blob/main/pyproject.toml#L57

Reply:
yeah that's weird to see all those merge conflicts.

Reply:
ok, I tried that strategy and re-pushed. How does it look?

Reply:
Lgtm